Demo/Sources/Settings/SettingsView.swift

```diff
@@ -33,7 +33,9 @@ private struct UrlCacheView: View {
 
     private func clearUrlCache() {
         URLCache.shared.removeAllCachedResponses()
-        updateCacheSize()
+        DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
```

Comment:
There's no way to be notified when `removeAllCachedResponses` has completed? Or to observer the cache or its size?

Reply:
Nope, not really. `URLCache` API is sadly quite limited.